source/features/clean-repo-sidebar.tsx

```diff
@@ -0,0 +1,31 @@
+import './clean-repo-sidebar.css';
+
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	// Hide empty meta if it’s not editable by the current user
+	if (!select.exists('.repository-content [aria-label="Edit repository metadata"]')) {
```

Comment:
I think this should be used instead

https://github.com/fregante/github-url-detection/blob/4840d85d31d59bfe71e884ef42c482fbfff2d955/index.ts#L490

Reply:
I would, but the function needs to be fixed first since the `data-tab-item` values were changed by GH, cf. https://github.com/sindresorhus/refined-github/pull/3796/files and https://github.com/sindresorhus/refined-github/pull/3795#discussion_r537577913

Reply:
Let’s leave a TODO at least 👍